featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -1116,3 +1117,20 @@ def is_greater(x):
 
     for name in expected_names:
         assert name in fm.columns
+
+
+def test_time_since_primitive_matches_all_datetime_types(pd_es):
```

Comment:
Since the `TimeSince` primitive is compatible with Dask entitysets, we should use the `es` test fixture here instead of the `pd_es` test fixture so that this gets tested with both pandas and Dask entitysets.

Reply:
Done, thanks for the comment!